preview/index.ts

```diff
@@ -2,7 +2,9 @@ import { readFileSync, writeFileSync } from 'fs';
 import { NonEmptyString } from 'io-ts-types';
 import { getEmailNewsletters, rowToNewsletter } from '../src/jobs/newsletters';
 import {
-	EmailNewsletter,
+	CancelledEmailNewsletterType,
+	CurrentEmailNewsletter,
```

Comment:
Does current here mean active?

Reply:
Kind of - it means 'not cancelled', so not necessarily being actively sent out , but not marked as permanently cancelled either.  It could be renamed to  `NotCancelledEmailNewsletter` or similar.

Reply:
I'd be in favour of naming `EmailNewsletter` to mean the non-cancelled type, and `CancelledEmailNewsletter` otherwise. I think `Current...` might be a bit confusing in some places, especially where there isn't a `Cancelled...` nearby

Reply:
That makes sense - I should take a look to see if we actually need a named union type (currently `EmailNewsletter` is `CancelledEmailNewsletter` |  `CurrentEmailNewsletter `), but if we can do without that I should be simpler